pkgs/development/python-modules/python-ldap/default.nix

```diff
@@ -53,7 +67,9 @@ buildPythonPackage rec {
   doCheck = !stdenv.isDarwin;
 
   meta = with lib; {
+    changelog = "https://github.com/python-ldap/python-ldap/releases/tag/python-ldap-${version}";
     description = "Python modules for implementing LDAP clients";
+    downloadPage = "https://github.com/python-ldap/python-ldap";
     homepage = "https://www.python-ldap.org/";
     license = licenses.psfl;
   };
```

Comment:
Is anyone in particular maintaining this? Would you be okay volunteering?

Reply:
I'm basically the fallback maintainer for all python packages anyway, so it wouldn't change anything.

Reply:
Good to know. Sorry about that again!

Reply:
No worries!